libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -75,9 +72,7 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
      * @return true if the raw location was usable, false if not.
      */
     override suspend fun updateLocation(rawLocation: Location, date: Date): Boolean {
```

Comment:
Can we move the dispatcher definition near MapboxNativeNavigatorImpl

It would be nice to be able to define each of these functions with something like `withContext(NativeDispatcher)`